packages/components/package.json

```diff
@@ -22,7 +22,7 @@
         "install:playwright": "playwright install --with-deps",
         "lint": "eslint .",
         "lint:fix": "eslint --fix .",
-        "storybook": "storybook dev -p 6007",
+        "storybook": "storybook dev -p 6006",
```

Comment:
what is the reason to switch the ports between the components and legacy fondue?